### DISCUSSION TITLE
好低级的项目呀，直接搭建一个FQ服务不就好了，多此一举

### DISCUSSION THREAD

**Discussion by googlechn** [SeaHOH/GotoX#171] [回收站]

**Comment by Sak94664** [comment]
其中伪装SNI的功能对于某些需要直接连线，不允许使用代理的网站，例如nyaa.si还是有实际作用的。这个项目也算是绕过SNI检查的一种实现，个人认为还是很有意义的。  
并且，~~怎么能这么说可爱的Sea酱~~

**Comment by ghost** [comment]
[Quote](https://web.archive.org/web/20160319231117/http://dlc.nekonazo.com/n11954.1/) :

> “在这个国度，很多人没有理想，并且以摧毁别人的理想为乐趣。”


这名未经授权(?)使用 Google 所有的 logo 的 GitHub 用户: 
建议您重修小学语文。

---

请再读一遍吧，以下内容 copy 自 GotoX 的 README.md 文件:

- GotoX 修改自 goagent，其主要目的在于，当访问的网络服务出现问题，用户可以通过 方便快速地添加更改规则来自行解决。
- 其特色，一是自动代理，可支持标准 HTTP/1.1 请求；二是可通过修改 TLS 的 SNI 扩展 直连大部分网站。
主要使用 CFWorkers 服务作为后端代理，也支持 HTTP/SOCKS4/SOCKS5 代理，两者处于 同等地位。SOCKS 代理支持认证。
- 配置 CFWorkers 代理仅限单个账户内，支持绑定的自定义域名。
- 运行时可一直维护一个较小但快速的 GWS IP 列表（用于访问谷歌网络服务）

---

GoAgent 之前很多人使用， GotoX 也是一种免费的反审查工具。好比之前的西厢计划， 提供 hosts 文件的 repo，Tor 浏览器的 meek 网桥，和 shadowsocks 不同的反审查软件。

就好像在 Apache 的 Mailing List 问为什么不用 Nginx 一样，

> 好 **低级** 的项目呀，直接搭建一个 **FQ** 服务不就好了，多此一举 

> What a low-level project, is it not enough to build an FQ service directly?

给不懂新话的解释: FQ 是 翻墙(Fan-Qiang, go over the wall) 的意思

当然了，更 「**好**」 的方法是直接购买 **FQ** 服务，您 「**搭建**」 : 配置 v2ray/ss 当然也是 **多此一举** 了。

更加 「**好**」 的方法是移民到可以使用您的头像的服务的国家。 于是所有 censorship-circumvention 都是 **多此一举** 了。

我不太明白您的 「低级」 是指什么。

如果是 「底层」: GotoX 的修改只在应用层
如果是道德上的 「低级」（低级趣味的-）: 对一个开源项目也不适用

但是毫无疑问的是： 您不喜欢这个项目。

在这种情况下，您可以: 

- 关闭这个项目的 Tab/窗口
- 不使用
- 删除这个 issue，避免以后其他人的回復将您带回这个 repo
- 向 Cloudflare 检举滥用行为
